linera-core/src/chain_worker/state/mod.rs

```diff
@@ -430,7 +430,7 @@ where
         &self,
         height_map: Vec<(Medium, Vec<BlockHeight>)>,
         recipient: ChainId,
-    ) -> Result<CrossChainRequest, WorkerError> {
+    ) -> Result<Option<CrossChainRequest>, WorkerError> {
```

Comment:
This is just an optimization, right?

Reply:
Actually, `None` should be unreachable. I'll change it back.